src/map/ai/ai_container.cpp

```diff
@@ -279,6 +289,13 @@ void CAIContainer::Internal_Raise()
         ForceChangeState<CRaiseState>(entity);
 }
 
+void CAIContainer::Internal_UseItem(uint16 targetid, uint8 loc, uint8 slotid)
+{
+    auto entity{ dynamic_cast<CBattleEntity*>(PEntity) };
+    if (entity)
+        ChangeState<CItemState>(static_cast<CCharEntity*>(entity), targetid, loc, slotid);
```

Comment:
i changed my mind and copied exactly what you did so when you change your mind you can change this too

get merging